packages/babel-preset-default/bin/index.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+require( 'core-js-builder' )( {
+	modules: [ 'es', 'web' ],
```

Comment:
It's based on:

https://github.com/zloirock/core-js/blob/67ad54d1f1c455c300e4757e72039d8b586b743c/scripts/generate-indexes.js#L26